.travis.yml

```
@@ -35,7 +35,7 @@ install:
       -v "$WIT_WORKSPACE:/mnt/workspace" \
       --workdir /mnt/workspace \
       "$WIT_DOCKER_IMAGE" \
-      bash -c "git config --global url.'https://github.com/'.insteadOf 'git@github.com:' && wit init /mnt/workspace -a /mnt/block-pio-sifive -a git@github.com:sifive/environment-blockci-sifive.git::0.3.0"
+      bash -c "git config --global url.'https://github.com/'.insteadOf 'git@github.com:' && wit init /mnt/workspace -a /mnt/block-pio-sifive -a git@github.com:sifive/environment-blockci-sifive.git::0.4.0"
```

Sorry again, we made in change in wit 0.13.0's Docker image to use a slimmed down Linux distribution named Alpine, rather than Ubuntu, and bash isn't installed by default. You can change this to:

```suggestion
      sh -c "git config --global url.'https://github.com/'.insteadOf 'git@github.com:' && wit init /mnt/workspace -a /mnt/block-pio-sifive -a git@github.com:sifive/environment-blockci-sifive.git::0.4.0"
```

/cc @mmjconolly on this, where unfortunately, since this is on Travis CI and not GitHub Actions, the fix that we made to install `bash` into the image won't work either, since that was to the GitHub Action Docker image, not the main Wit Docker image.

---

Ah I should have left the `bash` in both